templates/appliance/rocket-chat/_next-steps.html

```diff
@@ -0,0 +1,32 @@
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h2>
+        Start using your {{ appliance.name }} Ubuntu Appliance
+      </h2>
+      <p>
+        Now you can go ahead and set everything up.  It’s usually best to wait 5-10 minutes here. Some snaps will need to refresh themselves and the appliance may need to automatically reboot. When you’re ready go to:
+      </p>
+      <pre><code>http://&lt;IP-of-your-appliance&gt;:3000/web</code></pre>
```

Comment:
Shouldn't this use the data from the yaml?

Reply:
the entire text changes with each app, not sure it is worth it.

Reply:
done and updated the copy doc